skimage/util/_backends.py

```diff
@@ -0,0 +1,146 @@
+import functools
+from importlib.metadata import entry_points
+from functools import cache
+import os
+import warnings
+
+
+def _entry_points(group):
+    # Support Python versions before 3.10, which do not let you
+    # filter groups directly.
+    all_entry_points = entry_points()
+    if hasattr(all_entry_points, "select"):
+        selected_entry_points = all_entry_points.select(group=group)
+    else:
+        selected_entry_points = all_entry_points.get(group, ())
+    return selected_entry_points
```

Comment:
Should we now remove this function because now scikit-image has dropped the support for Python 3.10?
```suggestion
    return entry_points(group=group)
```

Reply:
Yes, that sounds sensible. Though, I think we can remove the extra function altogether since entry_points(group=group) is trivial. Addressed in https://github.com/scikit-image/scikit-image/pull/7520/commits/266345b1fad2684be695b944b155d81aadfd720a.